hedvig.config.js

```diff
@@ -9,5 +9,5 @@ module.exports = {
   port: 8038, // The WDS port
   developmentPublicPath: 'http://0.0.0.0:8038/', // Client public path during development, i.e. "http://0.0.0.0:8081/". Port must match the port directive
   productionPublicPath: undefined, //  Client public path in production, i.e. "/assets/"
-  envVars: ['USE_AUTH', 'AUTH_NAME', 'AUTH_PASS', 'GIRAFFE_ENDPOINT'],
+  envVars: ['USE_AUTH', 'AUTH_NAME', 'AUTH_PASS', 'getGiraffeEndpoint'],
```

Comment:
🤔

Reply:
hahahaha 😂 webstorm is a bit too eager sometimes hehe